label_sync/labels.yaml

```diff
@@ -22,19 +22,56 @@
 ---
 default:
   labels:
-    # Signals to other contributors
+    # Common labels for triaging
+    - color: c5def5
```

Comment:
Just curiosity, how are the colors being chosen?

Reply:
Most of the colors are chosen randomly :) while there is a general principle that all "positive" labels should be green but "negative" labels be red, like `lgtm` and `approve` are green or light green, but `hold` is red. And all "neutral" labels are random, ideally neither green nor red.

Reply:
Thanks for the explanation. Color codes tend to have cultural interpretations and therefore my interest.

Given the impossibility of having different concurrent codes right now and the importance of making it consistent, it sounds great from my side.